packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -313,7 +347,204 @@ const ToggleMenuDrawer = React.forwardRef(
                                     )}
                                 </MobileDrawer.Body>
                             )}
-                            {!is_appstore && (
+                            {is_pre_appstore && (
+                                <React.Fragment>
+                                    {is_logged_in && is_trading_hub_category ? (
+                                        <MobileDrawer.SubHeader
+                                            className={classNames({
+                                                'dc-mobile-drawer__subheader--hidden': is_submenu_expanded,
+                                            })}
+                                        >
+                                            <Button
+                                                className={classNames({
+                                                    'header__menu--back-to-ui': !is_dark_mode,
+                                                    'header__menu--back-to-ui--dark': is_dark_mode,
```

Comment:
```suggestion
                                                    `header__menu--back-to-ui${is_dark_mode ? '--dark' : ''}`
```
You can use string interpolation here since it is simpler